src/station.cpp

```diff
@@ -565,3 +629,35 @@ Money AirportMaintenanceCost(Owner owner)
 	/* 3 bits fraction for the maintenance cost factor. */
 	return total_cost >> 3;
 }
+
+void IndustryVector::Include(Industry *i)
+{
+	for (auto it = this->Begin(); it != this->End(); ++it) {
+		/* Do nothing further if industry is already in the list. */
+		if (*it == i) return;
```

Comment:
Why are these functions necessary? This operation makes them seem like a set, rather than a vector

Reply:
It was actually originally a std::set, however (at least for stations) the existing type was used elsewhere so there was an additional step to convert it every time.

Reply:
And now it is a std::set